geonetwork/content.md

```diff
@@ -12,50 +12,151 @@ The project is part of the Open Source Geospatial Foundation ( [OSGeo](http://ww
 
 %%LOGO%%
 
-# How to use this image
+## How to use this image
 
-## Start geonetwork
+GeoNetwork 4 uses an Elasticsearch server to store the index of the documents it manages so **it can't be run without configuring the URL of the Elasticsearch server**. This is a quick example of how to get GeoNetwork up and running for demo purposes. This configuration doesn't keep the data if containers are removed.
```

Comment:
```suggestion
GeoNetwork 4 uses an Elasticsearch server to store the index of the documents it manages so **it can't be run without configuring the URL of the Elasticsearch server**.

This is a quick example of how to get GeoNetwork 4.4 Latest up and running for demo purposes. This configuration doesn't keep the data if containers are removed.
```